src/grid/gridColumn.js

```diff
@@ -6,10 +6,9 @@ class GridColumnComponent extends React.Component {
         return !isEqual(this.props.children, nextProps.children);
     }
     render() {
-        const flexSize = this.props.flexGrow || '1';
-        const style = { flex: `${flexSize} 0 0px` };
+        const style = { flex: '1 0 0px' };
```

Comment:
The more constrictive approach would be to allow the user to set flex-shrink, flex-basis, etc. like flex-grow is currently being set. If I recall correctly, this was my first idea.

The problem with this is, if you let the user pass one variable (flex-grow), and then change it to let them pass another (flex-basis), why not just let them specify what they want in the first place (flex)? There are other flex properties that could be needed as well, but by limiting them to certain properties, and adding them as needed, the code will need to be updated repeatedly. This can be circumvented by allowing them to specify the styles they want.